### ISSUE TITLE
Can svn automatically fix up end of line characters for .C and .h files?

### ISSUE THREAD

**Issue by aowen87** [aowen87/BAR#2077]
I made some new files on Windows and I got errors when I tried to check them in. Can we set up svn to automatically convert the dos newlines to UNIX-style newlines? This would have no ill-effect on the files and would make Windows development friendlier. Guess what? No dos2unix on Mac or Windows so this is a real pain for me.

Transmitting file data ............svn: Commit failed (details follow):
svn: Commit blocked by pre-commit hook (exit code 1) with output:
File "branches/2.5RC/src/viewer/main/VisItChangeUsernameWindow.C" appears to contain '^M' characters, maybe from dos?.
Please remove them before committing. Try using dos2unix tool.

-----------------------REDMINE MIGRATION-----------------------
This ticket was migrated from Redmine. The following information
could not be accurately captured in the new ticket:

Original author: Brad Whitlock
Original creation: 06/13/2012 04:39 pm
Original update: 06/21/2012 10:58 am
Ticket number: 1096
